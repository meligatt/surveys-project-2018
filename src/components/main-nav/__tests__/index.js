import React from 'react';
import MainNav from '../index';

describe('MainNav', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <MainNav />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
