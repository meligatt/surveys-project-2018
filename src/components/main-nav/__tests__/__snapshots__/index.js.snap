// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MainNav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="main-nav__container"
>
        <a
                className="main-nav__logo"
                href="/"
        >
                Survey system
        </a>
        <ul
                className="main-nav__menu"
        >
                <li
                        className="main-nav__item"
                >
                        <Link
                                replace={false}
                                to="/"
                        >
                                Home
                        </Link>
                </li>
                <li
                        className="main-nav__item"
                >
                        <Link
                                replace={false}
                                to="/about"
                        >
                                About
                        </Link>
                </li>
        </ul>
</div>,
      "className": "main-nav",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <a
            className="main-nav__logo"
            href="/"
>
            Survey system
</a>,
          <ul
            className="main-nav__menu"
>
            <li
                        className="main-nav__item"
            >
                        <Link
                                    replace={false}
                                    to="/"
                        >
                                    Home
                        </Link>
            </li>
            <li
                        className="main-nav__item"
            >
                        <Link
                                    replace={false}
                                    to="/about"
                        >
                                    About
                        </Link>
            </li>
</ul>,
        ],
        "className": "main-nav__container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Survey system",
            "className": "main-nav__logo",
            "href": "/",
          },
          "ref": null,
          "rendered": "Survey system",
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li
                className="main-nav__item"
>
                <Link
                                replace={false}
                                to="/"
                >
                                Home
                </Link>
</li>,
              <li
                className="main-nav__item"
>
                <Link
                                replace={false}
                                to="/about"
                >
                                About
                </Link>
</li>,
            ],
            "className": "main-nav__menu",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  replace={false}
                  to="/"
>
                  Home
</Link>,
                "className": "main-nav__item",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Home",
                  "replace": false,
                  "to": "/",
                },
                "ref": null,
                "rendered": "Home",
                "type": [Function],
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Link
                  replace={false}
                  to="/about"
>
                  About
</Link>,
                "className": "main-nav__item",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "About",
                  "replace": false,
                  "to": "/about",
                },
                "ref": null,
                "rendered": "About",
                "type": [Function],
              },
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="main-nav__container"
>
          <a
                    className="main-nav__logo"
                    href="/"
          >
                    Survey system
          </a>
          <ul
                    className="main-nav__menu"
          >
                    <li
                              className="main-nav__item"
                    >
                              <Link
                                        replace={false}
                                        to="/"
                              >
                                        Home
                              </Link>
                    </li>
                    <li
                              className="main-nav__item"
                    >
                              <Link
                                        replace={false}
                                        to="/about"
                              >
                                        About
                              </Link>
                    </li>
          </ul>
</div>,
        "className": "main-nav",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <a
              className="main-nav__logo"
              href="/"
>
              Survey system
</a>,
            <ul
              className="main-nav__menu"
>
              <li
                            className="main-nav__item"
              >
                            <Link
                                          replace={false}
                                          to="/"
                            >
                                          Home
                            </Link>
              </li>
              <li
                            className="main-nav__item"
              >
                            <Link
                                          replace={false}
                                          to="/about"
                            >
                                          About
                            </Link>
              </li>
</ul>,
          ],
          "className": "main-nav__container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Survey system",
              "className": "main-nav__logo",
              "href": "/",
            },
            "ref": null,
            "rendered": "Survey system",
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <li
                  className="main-nav__item"
>
                  <Link
                                    replace={false}
                                    to="/"
                  >
                                    Home
                  </Link>
</li>,
                <li
                  className="main-nav__item"
>
                  <Link
                                    replace={false}
                                    to="/about"
                  >
                                    About
                  </Link>
</li>,
              ],
              "className": "main-nav__menu",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    replace={false}
                    to="/"
>
                    Home
</Link>,
                  "className": "main-nav__item",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Home",
                    "replace": false,
                    "to": "/",
                  },
                  "ref": null,
                  "rendered": "Home",
                  "type": [Function],
                },
                "type": "li",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    replace={false}
                    to="/about"
>
                    About
</Link>,
                  "className": "main-nav__item",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "About",
                    "replace": false,
                    "to": "/about",
                  },
                  "ref": null,
                  "rendered": "About",
                  "type": [Function],
                },
                "type": "li",
              },
            ],
            "type": "ul",
          },
        ],
        "type": "div",
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
